kubernetes-client-api/src/main/java/io/fabric8/kubernetes/client/http/HttpRequest.java

```diff
@@ -55,6 +55,8 @@ default Builder put(String contentType, String writeValueAsString) {
       return method("PUT", contentType, writeValueAsString);
     }
 
+    Builder put(String contentType, InputStream stream, long length);
```

Comment:
We might want to go for reusable `Builder method(String method, String contentType, ...` methods that can be called from default implementations of post or put. Basically the same pattern we're following for String bodies.